src/screens/ProfileScreen/index.js

```diff
@@ -621,12 +613,12 @@ const ProfileScreen = ({route}) => {
         style={styles.flatlistContainer}
         onScroll={handleScroll}
         ListFooterComponent={isFetchingList ? <ActivityIndicator /> : null}
-        onEndReach={handleOnEndReached}
         initialNumToRender={2}
         maxToRenderPerBatch={2}
         updateCellsBatchingPeriod={10}
         removeClippedSubviews
         windowSize={10}
+        onMomentumScrollEnd={(event) => fetchNextFeeds(event)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onEndReach` prop has been removed from the FlatList component and replaced with `onMomentumScrollEnd`. This change could potentially improve performance by only loading more data when the scrolling has ended, rather than continuously as the user scrolls. However, ensure that this change does not negatively impact the user experience by causing noticeable delays in loading new data.

```diff
- onEndReach={handleOnEndReached}
+ onMomentumScrollEnd={(event) => fetchNextFeeds(event)}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->